### DISCUSSION TITLE
Introduction: John Oliver

### DISCUSSION THREAD

**Discussion by baouroux** [life-itself/community#161] [Introductions]
Greetings to the community and here to introduce myself, I'm currently connecting narratives sensemaking, developmental psychology and complexity principles - running an organisational developmental consulting brand original founded in 2009 in London and Paris.

From the beginning of our client work, working with narratives has been a central thread, based on a social sciences and anthropology lenses on organisational complexity. I've worked closely with Dave Snowden and the Cynefin Framework for key clients.  

In addition to narratives, I've trained in developmental psychology and have been inspired by Integral Theory (Ken Wilber) - I'm currently writing a book on connecting sensemaking, developmental psychology, complexity principles and consciousness studies into a notion of 'developmental sensemaking'.

My professional website is:
- www.collectivenavigator.com

With a '2nd person' witnessing practice with video portraits:
- www.interiortruth.com

And a '1st person' art practice:
- www.tyler.world

**Comment by rufuspollock** [comment]
@baouroux welcome John /cc @theo-cox @shiweiart

**Reply by theo-cox** [reply]
Welcome @baouroux thanks so much for the introduction! I'm Theo, part of the team here at Life Itself. I head up Life Itself Labs, which is our research and consulting arm. Having been in consulting for a few years myself now your approach seems fascinating, and I'd love to learn more. Would you be open to a brief introductory call to connect and get to know one another properly?

**Comment by baouroux** [comment]
Hi Theo,

Thanks for your email...wonderful to connect, and have been tuning into a
few of your podcasts. Super impressed by the level of discussion and your
inputs into the dialogues.

Yes, delighted to arrange for a call.

There's a really nice coincidence, in that we live in the SW of France, and
we took a family trip to see Sylvie's recent performance at the Thénac Hub,
just a couple of weeks ago. Amazing to see. Rufus and Sylvie are planning
to come to visit us here too very soon.

For a call; next week, Tues/Weds/Thurs are fairly open for me, in case any
of those could work?

All the best,
John

PS, I'll reach out also on LinkedIn!
John Oliver
Mobile: +33 631 60 2994

**Reply by theo-cox** [reply]
Hi John,

Great, and thanks so much for the kind words! I'll drop you a private message on LinkedIn to arrange a time.

Best wishes,

Theo

**Comment by BoazBF** [comment]
Hey @theo-cox, @rufuspollock @Liyubov
Delighted to meet you all here - thanks you kindly Rufus for the invitation!

A few short words of introduction, I love joy, depth, multi-level thinking and the embodied experience of fractal perceptions and behaviours. I work as a clinical psychologist and contemplative, and slowly broadening my engagement towards social and political activism. I ordained as a Buddhist monk in my early 20's, and since have kept up a committed silent practice daily and annually in retreats. I've now built an NGO in Switzerland with a little about 25 collaborators, offering capacity building trainings in clinical trauma healing and community empowerment. I've worked for most major humanitarian organizations in conflict-affected regions and emergency relief.

Here's 

- my personal website: http://boazfeldman.com/
- my organizational website: https://neurosystemics.org/

I'm interested in furthering a systemitization of my method for resiliency and empowerment for social and political change in an upcoming PhD/PxD, and supporting the uprising and networking of communities for bottom-up political movement, heart-centered civil society movements and other such initiatives.

Looking forward to delving into the topic of community here with you all and getting to know you!

May it be for the benefit of all beings.

**Reply by rufuspollock** [reply]
@BoazBF very much welcome 😄 

One small item: can i suggest creating your introduction in an entirely new discussion thread named e.g. "Introduction Boaz Feldman" via https://github.com/life-itself/community/discussions/new?category=introductions - that way each person's intro is separate 😄

**Reply by rufuspollock** [reply]
I've moved this to a new discussion thread for you 😄 https://github.com/life-itself/community/discussions/168

**Comment by asimong** [comment]
Hello, Christmas/holiday greetings! I'm Simon Grant, and I was on the November residency in Bergerac with Theo and others, where I also met Rufus. My skill set is not specially easy to describe. I'm not a programmer as such, though I'd like to develop my skills there. A bit of a systems analyst perhaps, a bit of a user-centred HCI person, and a lot about standards — my own area has been technical interoperability in learning technologies, and in particular e-portfolios, skills and competence. I have a deep background in teaching as well, at secondary school (physics, maths, science) and university (HCI, systems analysis and design, e-commerce) levels. I love the meta-levels of things. Metadata, yes! Ontologies, yes! Specially when they are developed and owned inclusively and collaboratively.

Really, I'm interested in many projects that cross boundaries and silos between the information tech and the human tech; helping the information tech support what is of genuine value to humans; and working on the tech specifically to align better with human information processing. As I have been increasingly interested in the human, emotional psychological side of life, I'm have a strong interest in how we can help each other learn, grow, develop, overcome our unhelpful patterns, etc. etc.  And a specific interest in working with collectivity, collaboration, cooperation more than with isolated individuals. Though also I was briefly a hypnotherapist back in the 80's.

My (overfull, sorry) home page is at https://www.simongrant.org/home.html but my recent writings from this last year are here https://wiki.simongrant.org/doku.php?id=here and from 2021 here https://www.simongrant.org/d/

I would be delighted to have one-to-one conversations with anyone here, to scope out the areas of common interest and potential collaboration.

My current longings are for
- co-living with co-working on shared projects in Belgium (or UK, but would prefer to stay in Belgium working towards citizenship and freedom to move in EU
- developing knowledge commons and everything around that, including linking up various alternative efforts towards higher-level research, learning, education and training
- finding others keen enough to help me develop a prototype matching system, which I codename "CHOICE", which will contribute in many many ways to the economy and society of the future.

**Comment by baouroux** [comment]
Fascinating to read of your background and research Simon...many threads I
look forward to following up on in this group!
Best,
John

John Oliver
Mobile: +33 631 60 2994
www.interiortruth.com
ww.collectivenavigator.com
www.tyler.world
